DownFall/Extensions/CGPoint+Extension.swift

```diff
@@ -95,6 +108,8 @@ extension CGPoint {
             x = -that.width/2 + this.width/2 + xOffset
         case .right:
             x = that.width/2 - this.width/2 + xOffset
```

Comment:
These might need to be translated to their parents coordinates system is `this` is actually being place in `that`